### PR TITLE
23/user profile

### PR DESCRIPTION
**Pull request by niwinz**

### CODE REVIEW

File: backend/src/uxbox/emails.clj [hunk 1]
```diff
@@ -62,3 +62,11 @@
 (def password-recovery
   "A password recovery notification email."
   (emails/build ::password-recovery default-context))
+
+(s/def ::pending-email ::us/string)
```
Comment (hirunatan) [inline backend/src/uxbox/emails.clj]: don't we have a schema for a valid email address?

File: frontend/src/uxbox/main/ui/settings/change_email.cljs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    (st/emit! (du/request-email-change data))))
+
+(mf/defc change-email-form
+  [{:keys [locale profile] :as props}]
+  [:section.modal-content.generic-form
+   [:h2 (t locale "settings.change-email-title")]
+
+   [:span.featured-note
+    [:span.text
+     [:span "We’ll send you an email to your current email "]
```
Comment (hirunatan) [inline frontend/src/uxbox/main/ui/settings/change_email.cljs:60]: There are some untranslated texts here

File: frontend/src/uxbox/main/ui/settings/header.cljs [hunk 3]
```diff
@@ -16,22 +16,60 @@
...
+;; [:section.main-bar
+;;  [:nav
+;;   [:a.nav-item
+;;    {:class (when profile? "current")
+;;     :on-click #(st/emit! (rt/nav :settings-profile))}
+;;    (t locale "settings.profile")]
+;;   [:a.nav-item
+;;    {:class (when password? "current")
+;;     :on-click #(st/emit! (rt/nav :settings-password))}
+;;    (t locale "settings.password")]]]]))
```
Comment (hirunatan) [inline frontend/src/uxbox/main/ui/settings/header.cljs]: May this be deleted?

File: frontend/src/uxbox/main/ui/auth.cljs [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+(mf/defc auth
+  [{:keys [route] :as props}]
+  (let [section (get-in route [:data :name])
+        locale (mf/deref i18n/locale)]
+    [:*
+     [:& messages]
+     [:div.auth
+      [:section.auth-sidebar
+       [:a.logo i/logo]
```
Comment (hirunatan) [inline frontend/src/uxbox/main/ui/auth.cljs]: Should have a href here, pointing to home for example.

File: frontend/src/uxbox/main/ui/settings/change_email.cljs (hunk 2 above)
Comment (niwinz) [reply frontend/src/uxbox/main/ui/settings/change_email.cljs:60]: yes, becasue this is peding to be refactored in the notifications US

File: frontend/src/uxbox/main/ui/settings/header.cljs (hunk 3 above)
Comment (niwinz) [reply frontend/src/uxbox/main/ui/settings/header.cljs]: true

File: frontend/src/uxbox/main/ui/auth.cljs (hunk 4 above)
Comment (niwinz) [reply frontend/src/uxbox/main/ui/auth.cljs]: What is the home page in this example?

Comment (hirunatan) [reply frontend/src/uxbox/main/ui/auth.cljs]: I think the dashboard if the user is logged in, or the root if not. In fact, I think the / address should always redirect to dashboard when the user is logged in.

Comment (hirunatan) [reply frontend/src/uxbox/main/ui/auth.cljs]: But this is not an issue of this MR. What I'd do here is that the logo directs to /, and then see if we want this to go dashboard in other issue.

Comment (niwinz) [reply frontend/src/uxbox/main/ui/auth.cljs]: ok

**Pull request merged**